recipes/gstreamer/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.8.0" %}
+{% set sha256 = "947a314a212b5d94985d89b43440dbe66b696e12bbdf9a2f78967b98d74abedc" %}
+
+package:
+  name: gstreamer
+  version: {{ version }}
+
+source:
+  fn: gstreamer-1.8.0.tar.xz
+  url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win or osx]
```

Comment:
We just need this for Linux right now, Gstreamer is not needed by Qt5 on OS X.

However, we could try to make it work for OS X on the feedstock.